app/controllers/folders_controller.rb

```diff
@@ -23,6 +23,17 @@ def create
     end
   end
 
+  def destroy
+    @folder = Folder.find(params[:id])
+     if @folder.destroy
+       flash[:danger] = "Folder deleted."
+       redirect_to folio_path
+     else
+      flash[:danger] = "Folder not deleted."
+      redirect_to folio_path
+    end
```

Comment:
@JonathanSR I think I'll move this logic to Ashley's module once that's merged! Great work 👍